ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -539,7 +539,7 @@ export default defineComponent({
 <style lang="scss" scoped>
 @import '@/assets/styles/theme.scss';
 .sticky-container {
-  z-index: 7!important;
+  z-index: 4!important;
```

Comment:
To put the Fee Summary UNDER the loading overlay layer

Reply:
Yea, we definitely need to update `z-index` for Fee Summary, because in some cases (not necessary MHR) it blocks user menu (if it's long).

<img width="572" alt="Screen Shot 2022-12-07 at 11 37 23 AM" src="https://user-images.githubusercontent.com/2333290/206278936-bcedcc6d-eed9-477e-a721-7103419ba09a.png">

Reply:
Totally! This is fixed in our Transfer flow but i should make this global